### DISCUSSION TITLE
Renaming the default branch to `main`

### DISCUSSION THREAD

**Discussion by nicolasbock** [lanl/bml#503] [General]
A growing number of open source projects is considering to rename or has already renamed their default branch in git to `main`. GitHub for example has released guidance on the subject [1] and some further reading can be found here [2, 3].

Personally I am in favor of renaming `master` to `main`.

What is the general opinion here regarding the name of the default branch?

[1] https://github.com/github/renaming
[2] https://sfconservancy.org/news/2020/jun/23/gitbranchname/
[3] https://lore.kernel.org/git/pull.656.git.1591823971.gitgitgadget@gmail.com/

**Comment by jeanlucf22** [comment]
I would rather keep it as is now. I don't see a case for changing it.
